Solutions/Netskope/Package/mainTemplate.json

```diff
@@ -0,0 +1,602 @@
+        "version": 1
+      }
+    },
+    {
+      "type": "Microsoft.OperationalInsights/workspaces/providers/metadata",
+      "apiVersion": "2022-01-01-preview",
+      "name": "[concat(parameters('workspace'),'/Microsoft.SecurityInsights/',concat('Parser-', last(split(variables('_parserId1'),'/'))))]",
+      "dependsOn": [
+        "[variables('_parserId1')]"
+      ],
+      "properties": {
+        "parentId": "[resourceId('Microsoft.OperationalInsights/workspaces/savedSearches', parameters('workspace'), variables('parserName1'))]",
+        "contentId": "[variables('_parserContentId1')]",
+        "kind": "Parser",
+        "version": "[variables('parserVersion1')]",
+        "source": {
+          "kind": "Solution",
+          "name": "Netskope",
+          "sourceId": "[variables('_solutionId')]"
+        },
+        "author": {
+          "name": "Microsoft",
+          "email": "support@microsoft.com"
+        },
+        "support": {
+          "name": "Netskope",
+          "tier": "Partner",
+          "link": "https://www.netskope.com/services#support"
+        }
+      }
+    },
+    {
+      "type": "Microsoft.OperationalInsights/workspaces/providers/metadata",
+      "apiVersion": "2022-01-01-preview",
+      "properties": {
+        "version": "2.0.0",
+        "kind": "Solution",
+        "contentSchemaVersion": "2.0.0",
+        "contentId": "[variables('_solutionId')]",
+        "parentId": "[variables('_solutionId')]",
+        "source": {
+          "kind": "Solution",
+          "name": "Netskope",
+          "sourceId": "[variables('_solutionId')]"
+        },
+        "author": {
+          "name": "Microsoft",
```

Comment:
Hi @elforb - can you please refer this excel sheet and modify the author details in the main template 

https://microsoft.sharepoint.com/:x:/r/teams/AzureSecurityInsights-Ecosystem/_layouts/15/doc2.aspx?sourcedoc=%7B10a19238-43d6-4993-bba3-d797baa3108e%7D&action=edit&activeCell=%27Solutions%20Migration%20Tracker%27!A1&wdinitialsession=087e96a5-3708-412c-8ce3-6013fd0fd89e&wdrldsc=68&wdrldc=1&wdrldr=AccessTokenExpiredWarning%2CRefreshingExpiredAccessT